web/cobrands/sass/_base.scss

```diff
@@ -2499,61 +2537,71 @@ label .muted {
 }
 
 a#geolocate_link {
-  display: inline-block;
-  padding: 0.5em;
+  border: 1px solid $geolocation-link-border;
+  color: $geolocation-link;
+  @include button-size($button-margin-top: 15px);
+  svg {
+    display: inline-block;
+    vertical-align: text-bottom;
```

Comment:
FWIW these days, when it comes to vertically centering two horizontal siblings in an element (like an icon and some text in a button), I tend to use `display: flex; align-items: center;` on the parent, rather than faffing around with `vertical-align` like this. No matter what `vertical-align` you pick, it always ends up looking wrong at some font-size. At least `align-items: center` just does what it’s meant to.

We have some mixins that provide browser fallbacks for the flex properties – see the end of `_mixins.scss`.

Up to you whether you want to use it here. I think it’s probably worth it.